radixdlt/src/main/java/com/radixdlt/middleware2/network/MessageCentralValidatorSync.java

```diff
@@ -165,8 +174,17 @@ public void sendGetVerticesErrorResponse(BFTNode node, HighQC highQC) {
 					return null;
 				}
 
-				BFTNode node = BFTNode.create(src.getSystem().getKey());
-				return new GetVerticesErrorResponse(node, msg.highQC());
+				if (errorResponseRateLimiter.tryAcquire()) {
+					BFTNode node = BFTNode.create(src.getSystem().getKey());
+					return new GetVerticesErrorResponse(node, msg.highQC());
+				} else {
+					var counter = counters.add(CounterType.NETWORKING_DROPPED_ERROR_RESPONSES, 1);
```

Comment:
Should be increment(...) rather than add(..., 1).

Reply:
Fixed